doc/chomp_interface/scripts/collision_scene_test_2.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+import rospy
+from moveit_commander import RobotCommander, PlanningSceneInterface
+import geometry_msgs.msg
+import time
+
+class CreateScene(object):
```

Comment:
better class name. Why is this different from the above file?

Reply:
changed class name to ``CreateSparseScene`` should I keep the filename same as the class name or is this fine as is,  filename = ``collision_scene_example_sparse.py``?

Reply:
currently have class-name same as file-name